.github/workflows/build-publish-pr.yml

```diff
@@ -53,6 +54,51 @@ jobs:
           dockerhub_username: ${{ secrets.DOCKERHUB_READONLY_USERNAME }}
           dockerhub_password: ${{ secrets.DOCKERHUB_READONLY_PASSWORD }}
 
+      - name: Get PR labels
```

Comment:
This job won't run if the PR is already open and a label is added right? Will that cause issues? For example, if I forget to add the `crib` label before opening.

Reply:
Right, that's OK because the sync will also happen via label webhooks. The problem is if the label is added before the image is ready, the sync will be unhealthy. The goal with this is to run the sync as soon as the image is available.